terraform/modules/hub/files/matomo/matomo-adhoc-def.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "cpu": 512,
+    "memory": 512,
+    "essential": true,
+    "image": "${image_and_tag}",
+    "name": "matomo-adhoc",
+    "entryPoint": [
+      "/bin/sh",
+      "-c",
+      "tail -f /dev/null"
```

Comment:
So...what is the point of the adhoc task?

Reply:
I have no idea ... everything here was just lift-n-shifted from the existing deployment ... should we drop it?

Reply:
This adhoc task is used for extracting Matomo data from database. Performance analysts need to run SQL queries manually to see Matomo data and to download data.